protobuf-c/protobuf-c.c

```diff
@@ -3231,6 +3231,9 @@ protobuf_c_message_unpack(const ProtobufCMessageDescriptor *desc,
 	/* allocate space for repeated fields, also check that all required fields have been set */
 	for (f = 0; f < desc->n_fields; f++) {
 		const ProtobufCFieldDescriptor *field = desc->fields + f;
+		if (field == NULL) {
```

Comment:
@edmonds I guess checking `desc->fields` before the loop would be the more correct fix